app/src/renderer/connectors/api.js

```diff
@@ -10,7 +10,7 @@ const Client = (axios, remoteLcdURL) => {
   // returns an async function which makes a request for the given
   // HTTP method (GET/POST/DELETE/etc) and path (/foo/bar)
   function req(method, path) {
-    return async function(data) {
+    return async function (data) {
```

Comment:
linting issues?

Reply:
looks like it 😫

Reply:
eslint fix doesn't return any error with this

Reply:
I meant, there is a inconsistency in our eslint config we need to add a rule for.

Reply:
yep that's true, I was slowly adding the missing rules